TailLight/SetBlack.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+typedef struct _SET_BLACK_WORKITEM_CONTEXT {
```

Comment:
Would you mind using C++ style struct declaration, so that it becomes:
```
struct SET_BLACK_WORKITEM_CONTEXT {
    UNICODE_STRING symLink;
};
```

Reply:
Gentle reminder.

Reply:
Sure. Please verify that it's in.

Reply:
Thanks for explaining.